scripts/e2e-ci.sh

```diff
@@ -79,125 +82,140 @@ task() {
 }
 
 test_smoke() {
-    task InstructLab Smoke Test
+    task InstructLab smoke test
     ilab | grep --color 'Usage: ilab'
-    task InstructLab Smoke Test Complete
+    task InstructLab smoke test Complete
 }
 
-test_init() {
-    task Initializing ilab
-
-    step Setting train-profile for GPU accelerated training
+test_system_info() {
```

Comment:
What was the point of adding this test here besides that it was in `basic-workflow-tests.sh`? All we're doing here is seeing if the command doesn't crash, but that seems like something that would just go in a functional test run on every PR. 

If we searched the output for something that matched the T-shirt sizes GPU VRAM that might be useful. I know the command has some useful output, but I'd want to make sure we test that output too.

Reply:
Because the information is useful for debugging.